server/src/engineModel/EngineModel.ts

```diff
@@ -92,20 +93,17 @@ export class EngineModel {
                 this.completionProvider.addCompletionItem(method);
             });
         }
-        if (!!yclass.doc) {
-            ymlClass.setDocumentation(yclass.doc[0]);
-        }
+        const documentation: string = !!yclass.doc ? yclass.doc[0] : null;
+        ymlClass.setUserInformations(`(class) ${ymlClass.label}`, documentation);
         this.classes.push(ymlClass);
         this.enrichYmlClass(ymlClass);
         this.completionProvider.addCompletionItem(ymlClass);
     }
 
     private buildAttribute(attributeXmlElement: any, sourceType: string): YmlAttribute {
         const attribute = new YmlAttribute(attributeXmlElement.$.ident, this.uri);
-        attribute.detail = `[${sourceType}].${attribute.label}`;
-        if (!!attributeXmlElement.doc) {
-            attribute.setDocumentation(attributeXmlElement.doc[0]);
-        }
+        const documentation: string = !!attributeXmlElement.doc ? attributeXmlElement.doc[0] : null;
```

Comment:
I’m at like 8 % of the diff and I saw that `!!foo.doc ? foo.doc[0] : null` pattern thrice already. :exploding_head:
What about making it more readable and stuff via a function? You may even already have declared one for that somewhere, I dunno.